mungegithub/submit-queue/pv.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  labels:
+    app: submit-queue
+  name: submit-queue-github-cache
```

Comment:
this could just be named `submit-queue-pv`, since it can have more than just the github cache